static/src/javascripts/projects/common/modules/experiments/tests/next-in-series.js

```diff
@@ -64,6 +66,10 @@ define([
                         var el = $.create(render(series));
                         el.insertAfter($articleBody);
                     }
+                },
+
+                success: function(complete) {
+                    bean.on($('.next-in-series-test__remind-me-link')[0], 'click', complete);
```

Comment:
Will the tracking for this complete before the navigation completes? Not sure it matters, but curious!